src/autogpt_plugins/email/__init__.py

```diff
@@ -57,6 +58,10 @@ def post_prompt(self, prompt: PromptGenerator) -> PromptGenerator:
                 },
                 send_email_with_attachment,
             )
+        else:
+            print(
+                Fore.RED + f"{self._name} - {self._version} - Email plugin not loaded, because EMAIL_PASSWORD or EMAIL_ADDRESS were not set in env."
```

Comment:
As part of this, would you mind adding a utils folder with a logging file inside that exports a logger? Long term we will inject it into the plugins env but for now getting everyone off print seems like a good start

Reply:
The logger is acutally in auto-gpt. So we should use that one. But that is a discussion we still have to have - how we import autogpt here. autogpt is not a lib on pip or anywhere, so we can not easily import it in a supported way.